packages/web-components/src/components/rux-pop-up/rux-pop-up.scss

```diff
@@ -1,6 +1,9 @@
 :host {
     display: block;
     width: min-content;
+    font-size: var(--font-size-base); //prevents icon shift from font-size declaration on parent.
```

Comment:
can we move this to the internal `rux-popup` class? I just tried to quick and it seemed to work, were there any specific issues with this one?

Reply:
Yup, works great!